data/levels/christmas2016/worldmap.stwm

```diff
@@ -1,6 +1,6 @@
 (supertux-level
   (version 3)
-  (name (_ "Christmas 2016"))
+  (name (_ "Christmas Special"))
   (author "Carsten Wirtz")
```

Comment:
Were the name changes intended to be part of this PR? (They weren't mentioned in the PR desc)

There's also `Community Island 2020` -> `Community Island` and `Haloween 2014` -> `Halloween Special`.

Reply:
I renamed them because I originally had that check in place that would show both names from `info` and the name of the worldmap if the titles didn't match even if the level counts did. I removed that check now (that's what I wanted feedback about, but ultimately decided was pointless and removed it), so the titles don't need to be changed anymore. I still think it's better this way (the worldmap titles are consistent with the title in the `info` file), but if you think I should change them back, I will. These don't show up anywhere in the game anyway.

Reply:
As a matter of principle, it would be preferable to discuss add-on renamings with the community, just to ensure that the new name makes sense and that everybody is fine with it.